script/circleci

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+case $CIRCLE_NODE_INDEX in
+  0 )
+    bundle exec rake test
```

Comment:
> if this is the "good way" to do it on Circle, can we not use Travis now?

Yeah we could. I just tend to be despaired by Travis queueing, but maybe it's just me.